.github/workflows/tests.yml

```diff
@@ -118,14 +70,3 @@ jobs:
           path: |
             ./cypress/videos
             ./cypress/screenshots
-
-      - name: Codecov
```

Comment:
You can remove the one below, but not this one

Reply:
Ok, removed the "save coverage report" but not the "codecov-action". Can you explain the difference between them? @levity

Reply:
"Save coverage report" preserves the raw data--the same thing you would get in the `coverage/` folder if you ran `yarn test:ci` locally--as a Github Actions artifact.

"Codecov" uploads the lcov.info data file to Codecov so that you can see the report on their app.